source/core/ut_utils.pkb

```diff
@@ -798,5 +798,20 @@ create or replace package body ut_utils is
     return l_valid_name;
   end;
 
+  function add_prefix(a_list ut_varchar2_list, a_prefix varchar2, a_connector varchar2 := '/') return ut_varchar2_list is
```

Comment:
How about having add_prefix function for individual elements to avoid duplicating the below logic in 3 places:
```sql
        case 
          when self.is_anydata = 1 then
            l_prefix||'/'||trim (leading '/' from fl.column_value)
          else
            fl.column_value
          end
```

Reply:
good catch

Reply:
Actually that causes an issue due to SQL context and min grant user.
We will need to create a public synonym to utils